app/src/main/java/org/fossasia/openevent/app/login/LoginPresenter.java

```diff
@@ -1,26 +1,39 @@
 package org.fossasia.openevent.app.login;
 
 import org.fossasia.openevent.app.data.contract.ILoginModel;
+import org.fossasia.openevent.app.data.contract.IUtilModel;
 import org.fossasia.openevent.app.login.contract.ILoginPresenter;
 import org.fossasia.openevent.app.login.contract.ILoginView;
+import org.fossasia.openevent.app.utils.Constants;
+
+import java.util.Set;
 
 import javax.inject.Inject;
 
 public class LoginPresenter implements ILoginPresenter {
 
     private ILoginView loginView;
     private ILoginModel loginModel;
+    private IUtilModel utilModel;
 
     @Inject
-    public LoginPresenter(ILoginModel loginModel) {
+    public LoginPresenter(ILoginModel loginModel, IUtilModel utilModel) {
         this.loginModel = loginModel;
+        this.utilModel = utilModel;
     }
 
     @Override
     public void attach(ILoginView loginView) {
         this.loginView = loginView;
-        if(loginView != null && loginModel.isLoggedIn())
+
+        if(loginView == null)
+            return;
+
+        if(loginModel.isLoggedIn())
```

Comment:
If it is logged in, just call on success and return, no need to attach emails then